### PR TITLE
feat(sign_up): add style to sign up view

### PR DESCRIPTION
**Pull request by isavega**
image

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+<div class="flex min-h-full flex-col justify-center py-12 sm:px-6 lg:px-8">
+
+
+
+  <div class="sm:mx-auto sm:w-full sm:max-w-md">
+    <div class="mx-auto h-36 w-auto text-center text-8xl">
+      <img src="https://d2t1xqejof9utc.cloudfront.net/screenshots/pics/ccd583a310fc605f82c1f2345fadb0a7/large.png" class="object-contain h-36 w-full object-center" />
+    </div>
+      <h2 class="text-center text-5xl font-bold tracking-tight text-gray-900 uppercase mt-8">
+        Sign Up
```
Comment (iaheck) [inline app/views/devise/registrations/new.html.erb:11]: We should keep the original brand name here as in the login page
image

**Review by iaheck — CHANGES_REQUESTED** [review]
Since now we have a sign up page, we should put a button to get there from sign in as well

Only details, it looks great!

File: app/views/devise/registrations/new.html.erb (hunk 1 above)
Comment (isavega) [reply app/views/devise/registrations/new.html.erb]: image

image

**Review by iaheck — APPROVED** [review]

**Review by vpulgar — APPROVED** [review]
It looks perfect! 200 ok

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -1,29 +1,63 @@
...
+      </h2>
+  </div>
+
+
+  <div class="mt-2 text-center text-lg font-bold tracking-tight uppercase">
+    <p class="notice text-green-900"><%= notice %></p>
+    <p class="alert text-red-900"><%= alert %></p>
+  </div>
+
+<div class="mt-8 sm:mx-auto sm:w-full sm:max-w-md">
```
Comment (valentinjadot) [inline app/views/devise/registrations/new.html.erb:22]: usan Tailwind al final?

Comment (valentinjadot) [reply app/views/devise/registrations/new.html.erb:22]: Pensé que quedó en que no

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -1,29 +1,63 @@
...
+<div class="mt-8 sm:mx-auto sm:w-full sm:max-w-md"> 
+ <div class="bg-white py-8 px-4 shadow sm:rounded-lg sm:px-10">
+
+<%= form_for(resource, as: resource_name, :html => {:class => "space-y-6"}, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
   <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    <%= f.label :email, :class => "block text-sm font-medium text-gray-700" %><br />
```
[the remark this answers is not in the document]
Comment (valentinjadot) [reply app/views/devise/registrations/new.html.erb:29]: Better tu use class: "blabla" que :class => "blbla" which is old anotation

**Review by valentinjadot — CHANGES_REQUESTED** [review]
see comments above :)

**Review by marimaf — CHANGES_REQUESTED** [review]
There are empty lines that should be deleted.

**Review by valentinjadot — APPROVED** [review]

**Pull request merged**